### DISCUSSION TITLE
Welcome to hello-world Discussions!

### DISCUSSION THREAD

**Discussion by jdek-group** [jdek-group/hello-world#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jdek-group** [comment]
discussing how to discuss . @_@

**Comment by jaykay-20** [comment]
we can discuss here anything  right, i'm new to github pls bare with me

**Reply by jaykay-20** [reply]
please check the file if i'm going right or pls correct me

**Reply by hoyeehong** [reply]
@jaykay-20 yes the flow seems correct. But are you able to see your movie_df inside Postgres?
You might want to do a movie_df.read_sql() on your notebook to verify again

**Reply by jaykay-20** [reply]
@hoyeehong  thank you,  yes i'm able to see in postgres
